### PR TITLE
Drop debug output

### PR DESCRIPTION
**Pull request by ChrisBr**
https://paste.opensuse.org/view/raw/55501904

image

Thanks to @SergioAtSUSE for reporting.

### CODE REVIEW

File: src/api/app/views/webui2/shared/_repositories.html.haml [hunk 1]
```diff
@@ -31,7 +31,6 @@
       locals: { flags: flags[:useforbuild], project: project, package: package,
                 architectures: architectures, table_id: 'flag_table_useforbuild' }
 
-Hallo #{user_can_modify}
```
Comment (Ana06) [inline src/api/app/views/webui2/shared/_repositories.html.haml]: :rofl:

Comment (dmarcoux) [reply src/api/app/views/webui2/shared/_repositories.html.haml]: Let's put this in the funny fail of the year 2019.

**Comment by coolo** [comment]
this was a test for how careful @Ana06 reviews :)

**Comment by ChrisBr** [comment]
It was there for quite some time (25 days or something)... I remember I recognized it last week but then totally forgot about it :smile:

**Pull request merged**